model/span.go

```diff
@@ -95,6 +94,31 @@ func (s *Span) NormalizeTimestamps() {
 	}
 }
 
+// ParentSpanID returns ID of a parent span if it exists.
+// It searches for the first child-of reference pointing to the same trace ID.
+func (s *Span) ParentSpanID() SpanID {
+	for i := range s.References {
+		ref := &s.References[i]
+		if ref.TraceID == s.TraceID && ref.RefType == ChildOf {
```

Comment:
i've always wondered, why do we need traceid to be part of spanref? what use case is there when a span has a spanref to a different traceid?

Reply:
meta-traces; multi-trace joins in buffered writes